requirements.txt

```diff
@@ -1,4 +1,5 @@
 -r lint-requirements.txt
+https://github.com/kpu/kenlm/archive/master.zip
```

Comment:
@30stomercury I tried to run by installing with pip install kenlm and got some issues with `cython` and `gcc`. I installed the package via a git clone and pip install -e . and didn't get any issue. Have you installed it via pypi?

Reply:
Hi @johntremblay, please try 
```
pip install https://github.com/kpu/kenlm/archive/master.zip
```

Reply:
It worked thanks!